packages/web-components/fast-element/src/attributes.ts

```diff
@@ -78,6 +171,11 @@ export class AttributeDefinition implements Accessor {
         }
     }
 
+    /**
+     * Sets the value of the property on the source object.
+     * @param source - The source object to access.
```

Comment:
source object or HTML Element?

Reply:
They are one and the same in this particular case. I'll change it to `source element` to clarify.